client/styles/main.styl

```diff
@@ -72,9 +72,9 @@ nav
     text-indent -9999px
   for $app in $apps
     .{$app}
-      background url('images/'+ $app + '-logo.svg') 40% 50% no-repeat
+      background url('images/' + $app + '-logo.svg') 40% 50% no-repeat
 
-  .birt, .eidr
+  .birt, .eidr, .performance
```

Comment:
I followed the style that was in the file, but, sure, I can change it, no problem.